src/commands/fun/flipCoin.ts

```diff
@@ -0,0 +1,26 @@
+import { Message } from 'discord.js';
+import { ApplyOptions } from '@sapphire/decorators';
+import { Args, Command, CommandOptions } from '@sapphire/framework';
+import { BOT_PREFIX } from '../../bot';
+
+@ApplyOptions<CommandOptions>({
+  aliases: ['fc', 'flip', 'flip-coin', 'coin-flip', 'coinflip'],
+  description: 'Flip a coin! In making decisions, if it is not great, at least it is fair!',
+  detailedDescription: `**Examples:**\n\`${BOT_PREFIX}flip-coin\`\n
+  \`${BOT_PREFIX}fc\`\n
+  \`${BOT_PREFIX}flip\`\n
+  \`${BOT_PREFIX}coin-flip\`\n
+  \`${BOT_PREFIX}coinflip\`\n
+  \`${BOT_PREFIX}flipcoin\`
+  `
+})
+export class FunFlipCoinCommand extends Command {
+  async messageRun(message: Message, args: Args): Promise<Message> {
+    //Argument enforcement
+    if (!args.finished) {
+      return message.reply("`flipcoin` doesn't take any parameters");
```

Comment:
Do we need to enforce empty args? They're not needed but why not just run the command anyways?

Reply:
Oh hmmm, that's probably in response to https://github.com/uwcsc/codeybot/pull/176#discussion_r819705129 (screenshot below).
![image](https://user-images.githubusercontent.com/58180935/157688179-2a91c438-f58d-4c22-bcc9-b44cfd246903.png)
Currently, commands that take in 1+ arguments enforce number of arguments as well as type checking, while commands that take in 0 arguments do not enforce anything. As such, I guess it makes sense to me to also enforce 0 arguments (and then I had been planning to make a separate PR to enforce 0 arguments in the previous commands for consistency). Alternatively, what do you think about not enforcing number of arguments for any command? For example, `.bal word` and `.blackjack 50 word` would both work?

Reply:
Ah I see. If higher arg commands implicitrly enforce their arg numbers and counts, we might as well extend that behaviour for everything.